src/main/java/kg/jarkyn/cobspecserver/Runner.java

```diff
@@ -1,9 +1,23 @@
 package kg.jarkyn.cobspecserver;
 
+import kg.jarkyn.cobspecserver.middleware.ResponseController;
+import kg.jarkyn.cobspecserver.middleware.Router;
+import kg.jarkyn.cobspecserver.responders.FormResponder;
+import kg.jarkyn.cobspecserver.responders.OptionsResponder;
+import kg.jarkyn.cobspecserver.responders.PublicResourceResponder;
+import kg.jarkyn.cobspecserver.responders.RedirectResponder;
+import kg.jarkyn.cobspecserver.sockets.ListenerSocket;
+import kg.jarkyn.cobspecserver.utils.PublicResource;
+import kg.jarkyn.cobspecserver.utils.RequestParser;
+
 import java.io.IOException;
 import java.net.ServerSocket;
 
 public class Runner {
+    private static boolean KEEP_RUNNING = true;
+    private static int defaultPort      = 5000;
+    private static String publicResourcePath = "/Users/Jarkyn/Projects/8thLight/JAVA/cob_spec/public";
```

Comment:
It is of course possible, but wasn't a strict requirement to accept args.
I have a clear idea how I would do it, but I didn't include this in my estimates, so may not implement this before the submission.